app/src/main/java/com/mymeetings/android/utils/ClockUtils.kt

```diff
@@ -1,12 +1,46 @@
 package com.mymeetings.android.utils
 
 import android.text.format.DateUtils
-import android.util.TimeUtils
 import java.util.*
 
 
 class ClockUtils {
     fun currentTimeMillis() = System.currentTimeMillis()
 
-    fun getTimeLeft(timeInMillis : Long) = DateUtils.getRelativeTimeSpanString(timeInMillis)
+    fun getTimeLeft(timeInMillis : Long): String = DateUtils.getRelativeTimeSpanString(timeInMillis).toString()
+
+    fun getTimeInHoursAndMins(timeInMillis: Long): String {
+        val calendar = Calendar.getInstance()
+        calendar.timeInMillis = timeInMillis
+
+        val hour = calendar.get(Calendar.HOUR_OF_DAY)
+        val min = calendar.get(Calendar.MINUTE)
+        val am = calendar.get(Calendar.AM_PM)
+
+        val hourStr = if(hour < 10) {
```

Comment:
The logic to get formatted string for hours and minutes is duplicated, can move them into a private method.